src/main/java/mods/railcraft/common/blocks/aesthetics/cube/EnumCube.java

```diff
@@ -22,7 +22,7 @@
  */
 public enum EnumCube {
 
-    COKE_BLOCK(Module.FACTORY, "coke", new SimpleCube(), 2f, 10f),
+    COKE_BLOCK(Module.FACTORY, "coke", new FlammableCube(5, 300), 2f, 10f),
```

Comment:
Will fix in next commit. To prevent too many commits you could commit what I authored, and that feels better.